jenkins-builds/6.0.UP04-17.08-gpu

```diff
@@ -15,6 +15,7 @@
  Extrae-3.5.2-CrayIntel-17.08.eb
  GREASY-2.1-cscs-CrayGNU-17.08.eb                   --set-default-module
  GROMACS-2016.3-CrayGNU-17.08-cuda-8.0.eb           --set-default-module
+ GROMACS-2016.3-CrayGNU-17.08-PLUMED-2.3.0-cuda-8.0.61.eb  --set-default-module
```

Comment:
Is this a request from a single user? 

If this is for a single user, we should only provide him the easyconfig file and he should install it by himself.

Reply:
I removed the line again.